ambari-web/karma.conf.js

```diff
@@ -167,7 +168,7 @@ module.exports = function(config) {
     // - Opera (has to be installed with `npm install karma-opera-launcher`)
     // - Safari (only Mac; has to be installed with `npm install karma-safari-launcher`)
     // - IE (only Windows; has to be installed with `npm install karma-ie-launcher`)
-    browsers: ['PhantomJS'],
+    browsers: ['ChromeHeadless'],
```

Comment:
Make similar changes in other UI containing projects like ambari-admin, contrib\views\commons, contrib\views\files, contrib\views\capacity-scheduler

Reply:
Headless browser changes are done for ambari-admin by replacing phantomjs with chromeHeadless but in case of capacity-scheduler the use case of phantomjs is different and cannot be mapped to chromeHeadless so changes are done in capacity scheduler.

Reply:
ok